include/counter.h

```diff
@@ -75,8 +75,7 @@ struct counter_alarm_cfg {
 typedef void (*counter_wrap_callback_t)(struct device *dev, void *user_data);
 
 /* Deprecated counter callback. */
-typedef void (*counter_callback_t)(struct device *dev,
-						void *user_data);
+typedef void (*counter_callback_t)(struct device *dev, void *user_data);
```

Comment:
Why are you changing the deprecated callback in this commit? It seems like an unrelated change to the renaming of `counter_set_ch_alarm` to `counter_set_channel_alarm`

Reply:
yep, i will clean up a bit this PR since it became aggregation of couple of changes. Will split them into separate PRs